canvas_modules/common-canvas/src/object-model/pipeline-in-handler.js

```diff
@@ -240,6 +240,12 @@ export default class PipelineInHandler {
 								if (link.link_name) { // link_name is also optional
 									newLink.linkName = link.link_name;
 								}
+								if (link.type_attr) { // type_attr is also optional
+									newLink.attrType = link.type_attr;
```

Comment:
Is this correct? `attrType` below you have `typeAttr`

Reply:
Fixed.